less/index.less

```diff
@@ -57,3 +57,178 @@ html, body {
     height: 100%;
 }
 
+html {
+	font-size: 62.5%;
+}
+
+//Variables
+@pfont-size: 1.6rem;
+@row-color: #dddddd;
```

Comment:
Nice job creating some variables. Color variables are great for color scheming. Makes it really simple to change the look of a website, and you only need to change the color in one place!